docs/platforms/python/index.mdx

```diff
@@ -19,6 +19,12 @@ Don't already have an account and Sentry project established? Head over to [sent
 
 If you prefer to follow video instructions, see [How to Install the Sentry Python SDK in 60 Seconds](https://vimeo.com/899368672).
 
+<Note>
+
+Version 2.0 of the Sentry Python SDK is now available. If you're looking to upgrade from 1.x, check out the [migration guide](https://github.com/getsentry/sentry-python/blob/master/MIGRATION_GUIDE.md).
```

Comment:
Not sure exactly what the plans are for the migration guide, but should there's also an existing migration guide in the python docs here: https://docs.sentry.io/platforms/python/migration/ 

Will this new one replace the current one?

Reply:
I didn't know this existed! Will update that page too -- I probably wouldn't replace the content altogether but rather split it into two parts, one for upgrading from SDK version 1.x to 2.x, and one for the current content (upgrading from `raven-python` to `sentry-python`).

Reply:
Will do this in https://github.com/getsentry/sentry-docs/pull/9182